lib/commands/web.js

```diff
@@ -66,4 +66,24 @@ extensions.checkForAlert = async function () {
   return false;
 };
 
+extensions.waitForAtom = async function (promise) {
```

Comment:
ok I made requested changes. I will squash it once @imurchie see all changes and so that he understand @jlipps  comments.